src/main/java/io/papermc/hangar/controller/HangarController.java

```diff
@@ -41,10 +41,10 @@ protected ModelAndView fillModel(ModelAndView mav) {
         mav.addObject("rand", ThreadLocalRandom.current());
         mav.addObject("utils", templateHelper);
 
-        // alerts
-        if (mav.getModelMap().getAttribute("alerts") == null) {
-            mav.addObject("alerts", new HashMap<>());
-        }
+        // alerts // commented it out for now, so that it doesnt override the redirect's alerts
+//        if (mav.getModelMap().getAttribute("alerts") == null) {
+//            mav.addObject("alerts", new HashMap<>());
+//        }
```

Comment:
I don't get why this would override stuff, like the IF is there to make sure that doesnt happen

Reply:
Ah, forgot to tell that, my bad. 
When that if statement isnt comment out, and the redirect assigns an alert, the alert wont display. When that if statement is comment out, the alert will display.  

So I think that means the following: That the redirectattributes will only add stuff to the ModelAndView, when there isnt an attribute already. It will only check or it should add those redirectattributes after the path which it redirects to has been ran which mean that there already is an 'alerts' attribute because of that if statement.